spec/factories/live_content_item.rb

```diff
@@ -63,4 +63,10 @@
     sequence(:base_path) {|n| "/dodo-sanctuary-#{n}" }
     format "gone"
   end
+
+  factory :coming_soon_content_item, parent: :live_content_item do
```

Comment:
We've been naming these derived factories `x_live_content_item` as we frequently have draft equivalents.